scipy/stats/tests/test_stats.py

```diff
@@ -3639,8 +3633,9 @@ class SkewKurtosisTest:
     testmathworks = [1.165, 0.6268, 0.0751, 0.3516, -0.6965]
 
 
-@make_skip_xp_backends("skew")
+@make_skip_xp_backends(stats.skew)
 class TestSkew(SkewKurtosisTest):
+    @make_skip_xp_backends(stats.kurtosis)
     @pytest.mark.parametrize('stat_fun', [stats.skew, stats.kurtosis])
     def test_empty_1d(self, stat_fun, xp):
```

Comment:
Could you try moving this to `SkewKurtosisTest` and making `stat_fun` an attribute in each class? Would be better than applying `@make_skip_xp_backends(stats.kurtosis)` here, I think.